components/Auctions/index.js

```diff
@@ -0,0 +1,156 @@
+import Image from 'next/image';
+import React, { useState, useEffect } from 'react';
+import styles from './Auctions.module.css';
+import fetchData from '../../utils/fetchData';
+import fetchSelfDetails from '../../utils/fetchSelfDetails';
+
+const BASE_API_URL = process.env.NEXT_PUBLIC_BASE_API_URL;
+const AUCTIONS_URL = `${BASE_API_URL}/auctions`;
+const BASE_IMAGE_URL =
```

Comment:
Would it be better to have it in .env so that it can be used everywhere.... Just a thought

Reply:
@harshith-venkatesh Why not a file constant?

Reply:
That would be even better
I have observed this in status site, that we have gone with .env 
That's why I was biased with .env